station/clients/harbor_client.py

```diff
@@ -24,7 +24,7 @@ def get_artifacts_for_station(self, station_id: Union[str, int] = None) -> List[
             station_id = int(os.getenv("STATION_ID"))
         assert station_id
 
-        endpoint = f"/projects/station_{station_id}/repositories"
+        endpoint = f"/projects/station_{station_id}/repositories/?page_size=20"
```

Comment:
resolve pagination more generally

-> check if there are pages inside the response, if there are get all of them